e2e/docker/compose.yaml

```
@@ -7,28 +7,22 @@ services:
     ports:
       - "9091:9091"
       - "9092:9092"
-    command: [ "--insecure" ]
+      - "9093:9093"
+    command:
+      - --insecure
 
   # create a SuperNode service
   supernode:
-    build:
-      dockerfile: supernode.Dockerfile
+    image: docker.io/flwr/supernode:nightly
     deploy:
-      # specify 2 containers with 2xCPUs (for 2 clients)
+      # specify 2 containers with 2xCPUs
       replicas: 2
       resources:
         limits:
           cpus: '2'
-    command: [ "--insecure", "--superlink", "superlink:9092" ]
+    command:
+      - --insecure
+      - --superlink
+      - superlink:9092
     depends_on:
-      - superlink
-
-  # create a ServerApp service
-  serverapp:
-    build:
-      dockerfile: serverapp.Dockerfile
-    command: [ "--insecure", "--superlink", "superlink:9091" ]
-    # enforce dependency for graceful execution
-    depends_on:
-      - superlink
-      - supernode
+      - superlink
```

```suggestion
      - superlink

```

Add a missing line at the end of the file.